website/docs/developer/research/research-s3-bucket-policies.md

```diff
@@ -0,0 +1,158 @@
+
+## Reports Bucket
+
+Suggested name: PREFIX-orca-reports
+
+This bucket is relatively simple, as much of the interaction is automated through AWS. This bucket stores S3 Inventory reports generated by AWS, to be pulled in by `get_current_archive_list`.
+Versioning is not used, and in general no changes are required beyond updating the Permissions to the policy below:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Cross Account Access",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::909121343565:root"
+      },
+      "Action": [
+        "s3:GetObject*",
+        "s3:GetBucket*",
+        "s3:ListBucket",
+        "s3:PutObject",
+        "s3:PutObjectAcl",
+        "s3:PutBucketNotification"
+      ],
+      "Resource": [
+        "arn:aws:s3:::PREFIX-orca-glacier-archive-versioned",
+        "arn:aws:s3:::PREFIX-orca-glacier-archive-versioned/*"
+      ]
+    },
+    {
+      "Sid": "Inventory-PREFIX-orca-glacier-archive-versioned",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "s3.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::PREFIX-orca-glacier-archive-versioned/*",
+      "Condition": {
+        "StringEquals": {
+      	  "s3:x-amz-acl": "bucket-owner-full-control",
+      	  "aws:SourceAccount": "782417781503"
+        },
+      	"ArnLike": {
+      	  "aws:SourceArn": "arn:aws:s3:::PREFIX-orca-glacier-archive-versioned"
+      	}
+      }
+    },
+    {
+      "Sid": "Inventory-PREFIX-orca-glacier-archive-worm",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "s3.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::PREFIX-orca-glacier-archive-worm/*",
+      "Condition": {
+        "StringEquals": {
+      	  "s3:x-amz-acl": "bucket-owner-full-control",
+      	  "aws:SourceAccount": "782417781503"
+        },
+      	"ArnLike": {
+      	  "aws:SourceArn": "arn:aws:s3:::PREFIX-orca-glacier-archive-worm"
+      	}
+      }
+    }
+  ]
+}
+```
+The Principal value is the AWS root user for your Cumulus application that will
+access the ORCA archive bucket.
+
+Replace the number in `arn:aws:iam::909121343565:root` with the value of your account number.
+
+Replace the number `782417781503` with your DR account number.
+
+The Resource value is the bucket and bucket paths that the Cumulus application
+can access. Replace `sandbox-orca-glacier-archive` with the name
+of the Orca archive bucket created in the [Versioned Glacier Bucket](#versioned-glacier-bucket) section.
+
+It may be possible to combine the two `Inventory-PREFIX-orca-glacier-archive-*` rules, but this has not yet been tested.
+
+We should also remove the ACL capabilities due to AWS potential deprecation.
+This will be detailed in a [future section](#acl-rule-replacement).
```

Comment:
It would be good to also talk about a couple of additional aspects with the reporting bucket.
- Policy to delete reports older than x days since old reports are not useful once a new one is generated
- Potentially using tiered storage for cost savings if retention is long to save storage costs if a user wants to keep reports for an extended period of time

Reply:
Added notes on lifecycle rules.

Reply:
Added note on storage classes.